core/embed/rust/src/micropython/runtime.rs

```diff
@@ -1,10 +1,90 @@
-use cstr_core::CStr;
+use core::mem::MaybeUninit;
 
-use super::ffi;
+use crate::error::Error;
 
-pub fn raise_value_error(msg: &'static CStr) -> ! {
+use super::{ffi, obj::Obj};
+
+pub fn raise_exception(err: Error) -> ! {
```

Comment:
Nit, I'd rather make this unsafe.

Reply:
fixed in b406bd0a0ff986392c2fccef336a7b9201734023